docs/jobStore.rst

```diff
@@ -0,0 +1,21 @@
+The job store interface
+=======================
+
+Implementing the job store interface tutorial
+*********************************************
+
+The job store interface is used by Toil to abstract over different file stores, 
```

Comment:
How about something like 
`The job store interface is an abstraction layer that that hides the specific details of file sytem storage`

I think 'abstract over' is too.. abstract.